Sources/SwiftKuery/ConnectionPoolConnection.swift

```diff
@@ -56,19 +72,37 @@ public class ConnectionPoolConnection: Connection {
         }
     }
     
-    /// Close the connection to the database.
+    /**
+     Close the connection to the database.
+     ### Usage Example: ###
+     In this example, The closeConnection() function is called on a `ConnectionPoolConnection` instance called "connection" to release it from it's `ConnectionPool` instance.
+     ```swift
+     connection.closeConnection()
+     ```
+     */
     public func closeConnection() {
         if let connection = connection {
             pool?.release(connection: connection)
             self.connection = nil
         }
     }
     
-    /// An indication whether there is a connection to the database.
+    /**
+     An indication whether there is a connection to the database.
+     ### Usage Example: ###
+     In this example, The isConnected() function is called on a `ConnectionPoolConnection` instance called "connection" and returns a true value, which is then printed.
```

Comment:
Reword to - "In this example, the `isConnected()` function is called on a `ConnectionPoolConnection` instance called `connection`, this returns a true value that is then printed."